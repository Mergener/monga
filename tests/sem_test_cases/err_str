function foo() {
    var s : int;

    s = "Hello, world!";
}