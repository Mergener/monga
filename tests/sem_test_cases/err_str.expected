[Error] Module tests/sem_test_cases/err_str, line 4, column 25: Cannot assign value of type 'string' to lvalue of type 'int'.
Semantic errors were caught during analysis.
